tests/markdown/README.md

```diff
@@ -388,7 +388,7 @@ and add this config to your `package.json`:
   "lint-staged": {
     "*.{js,json,css}": [
       "prettier --write",
-      "git add"
+      "git add" // Warning: This breaks staging specific lines. See footnote below.
```

Comment:
You don't want to update this file. @ikatyang maybe we should rename it so people don't accidentally find it :P